onnxruntime/core/platform/windows/env.cc

```diff
@@ -227,6 +227,33 @@ class WindowsEnv : public Env {
     return telemetry_provider_;
   }
 
+  // \brief returns a value for the queried variable name (var_name)
+  std::string GetEnvironmentVar(const std::string& var_name) const override {
+    // Why getenv() should be avoided on Windows:
+    // https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/getenv-wgetenv
+    // Instead use the Win32 API: GetEnvironmentVariableA()
+
+    // Max limit of an environment variable on Windows including the null-terminating character
+    constexpr DWORD kBufferSize = 32767;
```

Comment:
Do we need to define this each time this function is called?

Reply:
No - can be static

Reply:
Resolving per offline discussion.